templates/deactivation-feedback-modal.php

```diff
@@ -20,8 +20,30 @@
 	$reasons_list_items_html = '';
 
 	foreach ( $reasons as $reason ) {
-		$list_item_classes = 'reason' . ( ! empty( $reason['input_type'] ) ? ' has-input' : '' );
-		$reasons_list_items_html .= '<li class="' . $list_item_classes . '" data-input-type="' . $reason['input_type'] . '" data-input-placeholder="' . $reason['input_placeholder'] . '"><label><span><input type="radio" name="selected-reason" value="' . $reason['id'] . '"/></span><span>' . $reason['text'] . '</span></label></li>';
+		$list_item_classes    = 'reason' . ( ! empty( $reason['input_type'] ) ? ' has-input' : '' );
+
+		if ( isset( $reason['extra_message'] ) && ! empty( $reason['extra_message'] ) ) {
```

Comment:
@fajardoleo same here, as in the other places, refactor from "extra message" to "internal message".